packages/next-on-pages/templates/_worker.js/utils/fetch.ts

```diff
@@ -32,6 +32,10 @@ function applyPatch() {
 	};
 }
 
+async function doImport(m: string) {
+	return import(m);
+}
```

Comment:
I'd probably have a single `doImport` function (maybe in its own file under `utils/`) so that we can also add comments to it without repetition, etc..